### PR TITLE
Kaydunov.  JH1T1-objects is compiled.

### PR DESCRIPTION
**Pull request by GardDen**

### CODE REVIEW

**Review by shpaser2 — APPROVED** [review]
lgfm

File: homework/english/kaydunov/lecture1/Present_continues.md [hunk 1]
```diff
@@ -0,0 +1 @@
+#Present_continues
```
Comment (AlexMackevich) [inline homework/english/kaydunov/lecture1/Present_continues.md:1]: ok

File: homework/english/kaydunov/lecture1/task1.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#Task1
+
+Hello JavaGuru!
```
Comment (AlexMackevich) [inline homework/english/kaydunov/lecture1/task1.md:3]: ok

File: homework/english/kaydunov/lecture1/task2.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#task2
+
+Some text.
+
+=End=
```
Comment (AlexMackevich) [inline homework/english/kaydunov/lecture1/task2.md:5]: ok

File: homework/java/kaydunov/lecture1/task1.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+- сложность днк
+- количество зараженных
+- смертность
+- заразность
+- количество убитых\
+:передаться( вероятность успеха передачи зависит от заразности; количество зараженных++)\
+:убить(зависит от смертности; количество убитых++; количество зараженных--)\
+:мутировать(сложность днк++; зарахность++;смертность++)
+
+
```
Comment (AlexMackevich) [inline homework/java/kaydunov/lecture1/task1.md:82]: ok

**Review by AlexMackevich — APPROVED** [review]

**Review by AlenaPalasionak — APPROVED** [review]
lgfm

**Pull request merged**